sources/packages/web/src/components/students/modals/PDStatusApplicationModal.vue

```diff
@@ -1,20 +1,32 @@
 <template>
   <modal-dialog-base
-    title="Apply for a permanent disability status"
+    title="Apply for a Disability Status"
```

Comment:
I believe that the `modal-dialog-base` can be replaced by the `confirm-modal` with this change.

Reply:
I was going towards confirm-modal approach initially, but considering this model to be temporary and going  to go back to it's previous state I took this approach of keeping modal-dialog-base. Let me know your thoughts